src/components/ProductsTable/ProductsTable.tsx

```diff
@@ -163,6 +183,11 @@ const ProductsTable: FunctionComponent<ProductsTableProps> = ({ data, isFetching
                 </Td>
                 <Td dataLabel={columnNames.sku}>{datum.sku}</Td>
                 <Td dataLabel={columnNames.quantity}>{datum.quantity}</Td>
+                <Td dataLabel={columnNames.serviceLevel}>{datum.serviceLevel}</Td>
+                <Td dataLabel={columnNames.unitOfMeasure.name}>
+                  {datum.unitOfMeasure?.name ?? <Text style={{ color: 'red' }}>Not Available</Text>}{' '}
```

Comment:
We didn't intend for "Not Available" to be in red.  that was just in the AC to highlight it during grooming.

Reply:
the red text in the UI makes it look like there is something wrong and the user could do something about it.  that's not the case.